### PR TITLE
Follow the project associated with current buffer

### PR DESCRIPTION
**Pull request by katspaugh**
See #36

### CODE REVIEW

**Comment by katspaugh** [comment]
@sabof I've removed `pe/open` and `pe/project-cache-filename`.

File: project-explorer.el [hunk 1]
```diff
@@ -1736,6 +1742,45 @@ outside of the project's root."
...
+  "Switch the project staying in the same window.
+If project-explorer isn't opened or no project associated with
+the current buffer, stop following."
+  (if (pe/get-project-explorer-window)
+      (when (or (pe/get-current-project-explorer-buffer)
+                (pe/has-cache))
+        (let ((win (selected-window)))
+          (project-explorer-open)
+          (select-window win)))
+    (pe/stop-follow-current)))
```
Comment (sabof) [inline project-explorer.el]: You probably want to check `pe/follow-current`, and not `(pe/get-project-explorer-window)` when disabling. What if a user has multiple frames?

Comment (katspaugh) [inline project-explorer.el]: @sabof right, I didn't think about multiple frames!

**Comment by katspaugh** [comment]
OK, I added the `pe/follow-current` check, but also left the `(pe/get-project-explorer-window)` check intact to avoid opening project explorer when it's not already opened in the current frame. 

Also squashed commits into one.

**Comment by sabof** [comment]
Seems to work, merged

**Comment by katspaugh** [comment]
Thanks!

**Comment by jimeh** [comment]
:heart: :smile:

**Pull request merged**